packages/sdk/identity/src/odis/phone-number-identifier.ts

```diff
@@ -29,6 +29,7 @@ export interface PhoneNumberHashDetails {
   e164Number: string
   phoneHash: string
   pepper: string
+  unblindedSignature?: string
```

Comment:
From what I can tell this shouldn't be optional, since objects of this type only originate from `getPhoneNumberIdentifierFromSignature`

Reply:
I found that Valora often uses this interface in their codebase just when passing variables around, so I didn't want to break that.

Reply:
I saw that `PhoneNumberHashDetails` is used a lot throughout Valora when passing data between functions - I didn't want to break all of that code

Reply:
I saw that Valora uses `PhoneNumberHashDetails` frequently throughout its codebase, didn't want to break their code

Reply:
Good point! If we were confident that they aren't instantiating any objects of type PhoneNumberHashDetails outside of getPhoneNumberIdentifierFromSignature then I believe we could safely make this field mandatory. That being said, I agree it might be safest to leave it optional if we're not confident there. I'll trust your judgement on that :)